File: test_vulnerable_code.py
# Test vulnerable Python code
password = "hardcoded_secret_123"
api_key = "sk-1234567890abcdef"

def unsafe_query(user_input):
    query = "SELECT * FROM users WHERE name = '%s'" % user_input
    return execute(query)

def render_user_content(content):
    from flask import render_template_string
    return render_template_string("<h1>" + content + "</h1>")

def process_user_code(code):
    result = eval(code)
    return result

File: test_vulnerable_code.js
// Test vulnerable JavaScript code
const apiKey = "sk-1234567890abcdef";
const password = "hardcoded_password_123";

function displayUserContent(userInput) {
    document.getElementById('content').innerHTML = userInput;
}

function executeUserCode(code) {
    return eval(code);
}

function processData(data) {
    // Potential XSS vulnerability
    document.body.innerHTML = "<div>" + data + "</div>";
}

